src/DocumentDB.ChangeFeedProcessor/FeedProcessing/PartitionProcessor.cs

```diff
@@ -66,9 +66,9 @@ public PartitionProcessor(IChangeFeedObserver observer, IChangeFeedDocumentClien
                     switch (docDbError)
                     {
                         case DocDbError.PartitionNotFound:
-                            throw new PartitionNotFoundException(requestContinuation);
+                            throw new PartitionNotFoundException("partition not found", requestContinuation);
```

Comment:
For consistency, it's better to capitalize 1st word and add period '.' in the end.